pkg/cli/cvd.go

```diff
@@ -27,22 +27,22 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
-type CVD struct {
+type RemoteCVDLocator struct {
 	ServiceRootEndpoint string `json:"service_root_endpoint"`
 	Host                string `json:"host"`
 	Name                string `json:"name"`
 }
 
-type CVDInfo struct {
-	CVD
+type RemoteCVD struct {
```

Comment:
If we never have "local CVDs" (which I don't think make sense in this context) adding "Remote" to the names is redundant.

Reply:
Used the `Remote` prefix to differentiate with the `CVD` API object, I understand these are in different modules but it makes it clearer.